modules/shared-memory.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-ephemeral-storage.adoc
+
+[id="shared-memory_{context}"]
+= Shared Memory
+
+
+Shared memory is one of the fastest IPC (Inter-process communication) mechanisms in Linux. There are two types of shared memory objects: System V and POSIX.
+Since the containers in a pod share the IPC namespace of the pod infrastructure container, they can automatically share the System V shared memory objects.
```

Comment:
s/Since/Because According to the IBM Style Guide use _since_ only in reference to time.

Reply:
Addressed